run/src/main/kotlin/com/cosmotech/run/service/RunServiceImpl.kt

```diff
@@ -280,8 +281,8 @@ class RunServiceImpl(
   ): RunData {
 
     checkInternalResultDataServiceConfiguration()
-    val run = getRun(organizationId, workspaceId, runnerId, runId)
```

Comment:
If this is really not necessary then just remove the code, don't comment it. But I somehow doubt we want to remove RBAC checks. (Although maybe this is done elsewhere now).

Same a few lines below for another `getRun`